test/runtime/samples/input-crazy-value-attribute/main.svelte

```diff
@@ -0,0 +1,5 @@
+<script>
+	export let value;
+</script>
+
+<input type="text" {value} />
```

Comment:
i dont think this will test whatever you are trying to test, since, there's the props are passed in during runtime, and the changes you made is on the parser which runs in the compile time.

i dont think we can have a test case over here to test how fast the parsing is, but we can test the correctness of the changes.

maybe can have a test case that has all the possible whitespace characters in between attribute values and make sure the parser can parse it correctly.

anyway since this change is more of a refactoring / behavior change, if existing test case does not break then the change should be safe to make.

Reply:
Thank you for clearing this up, @tanhauhau! I can create a test with all possible white space characters in between if you want. Should i do it?

Reply:
yup that would be great. I'll probably run the test before and after the change, and see if the change introduce any  regression

Reply:
I am wondering if this test will be a little weird, because it seems like the whitespace characters that were in the original RegEx are invalid whitespace characters for attribute values. So I'm not sure if this test will make sense. Please correct me if I'm wrong @tanhauhau

Reply:
its okay to be without adding new tests. since this PR is not fixing any bugs or changing any behavior